classes/author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
+  attr_accessor :first_name, :last_name
+  attr_reader :items
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..1000)
```

Comment:
The id is already obtained through the parent-class. Kindly consider omitting this line of code.

Reply:
Please create a separate directory inside the classes for more clarity.